resources/migrations/000_migrations.yaml

```diff
@@ -5275,3 +5275,80 @@ databaseChangeLog:
             columns:
               - column:
                   name: parent_id
+  - changeSet:
+      id: 102
+      author: sbelak
+      changes:
+        - createTable:
+            tableName: domain_entity
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: source_table
+                  type: int
+                  constraints:
+                    nullable: false
+                    references: metabase_table(id)
+                    foreignKeyName: fk_domain_entity_ref_source_table
+                    deferrable: false
+                    initiallyDeferred: false
```

Comment:
I'm 95% `deferrable: false` and `initiallyDeferred: false` are defaults anyway so you don't need to specify those.

Also you should consider adding an `ON DELETE CASCADE` to the FK constraint so if the Database gets deleted, and its Tables get deleted, the domain entity will get deleted automatically without having to use a Toucan `pre-delete` method to do it